File: app/src/main/java/ly/pp/justpiano3/entity/OLNote.java
package ly.pp.justpiano3.entity;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * 音符 缓存在队列用于传输
 */
@Data
@AllArgsConstructor
public class OLNote {

    /**
     * 时间
     */
    private long absoluteTime;

    /**
     * 音高
     */
    private int pitch;

    /**
     * 音量
     */
    private int volume;
}

File: app/src/main/java/ly/pp/justpiano3/entity/OLNote.kt
package ly.pp.justpiano3.entity

/**
 * 音符 缓存在队列用于传输
 */
data class OLNote(
    /**
     * 时间
     */
    var absoluteTime: Long,

    /**
     * 音高
     */
    var pitch: Int,

    /**
     * 音量
     */
    var volume: Int
)